package/MDAnalysis/analysis/lineardensity.py

```diff
@@ -109,14 +140,16 @@ def __init__(self, select, grouping='atoms', binsize=0.25, **kwargs):
         self.nbins = bins.max()
         slices_vol = self.volume / bins
 
-        self.keys = ['pos', 'pos_std', 'char', 'char_std']
+        self._keys = ['pos', 'pos_std', 'char', 'char_std']
+        # Create an alias for the results object to save writing
```

Comment:
What do you mean by "save writing"? Reducing IO operations, or you just want to re-use the old code in some way? There used to be a `self.results` object, but there is now a new `_Results` instance stored in `self.results`. You're saying that using `self._results` allows you to use more concise code than working with the instance directly, or to make less changes to the old code?

Reply:
The latter one. I want to make as less changes to the old code as possible. If this is confusing I can also remove this attribute.